src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+class Tribe__Events__Integrations__WPML__Meta {
```

Comment:
Another thing to add here is if you can add the `@since TBD` on the class, members and functions so we can add the version number when this changes has been introduced such as: 

```
/**
 * Some comment 
 *
 * @since TBD
 */
```

The `TBD` is changed by us when packaging for the next release. It's usually placed after the function comment, and above params or returns docs.

Reply:
Thanks for the pointer, I added them throughout the class.